dags/fivetran_acoustic.py

```diff
@@ -10,6 +10,9 @@
 
 
 def _generate_acoustic_report(conn_id, report_type, config, *args, **kwargs):
+    if config["request_params"]["date_start"] == config["request_params"]["date_end"]:
```

Comment:
Do you want to raise an exception or use a default in case any of the "date_start" or "date_end" keys is missing? Or is it maybe handled already in the previous step?

Reply:
You want a complete failure or a specific exception raised with the info?